cmu_graphics/cmu_graphics.py

```diff
@@ -360,6 +364,9 @@ def redrawAll(self, screen, cairo_surface, ctx):
 
         # Show PyGame surface
         screen.blit(pygame_surface, (0,0))
+        pygame.display.flip()
```

Comment:
This seems like an important function to call, from my understanding of the docs. How was it working without calling it before?